config/wallet-config.json

```diff
@@ -91,5 +91,5 @@
   },
   "recoverUninscribedTaprootUtxosFeatureEnabled": true,
   "runesEnabled": true,
-  "swapsEnabled": false
+  "swapsEnabled": true
```

Comment:
I thought that if I enable this it should in turn enable the swaps button but it doesn't. 

Do I need to do something else? @kyranjamie @fbwoolf ? 🤔 

`useConfigSwapEnabled` was moved to the mono-repo so I'm not sure if that affects it? 

If I change something else here e.g. remove an `activeFiatProvider` it is working

Reply:
Not sure here tbh, it looks like @edgarkhanzadian moved the code to the mono repo and is passing a `localConfig`. You might have to do some digging as to why it is broken?

I'm not seeing the code in the file changes that actually updates the alex sdk?

Reply:
OK thanks, I will dig more into it. 

Apologies, I must have accidentally committed adding the old SDK when I was checking if that was working. 

I will update it again now and dig more into this tomorrow / soon. I was hoping for a quick win but it seems like the new API doesn't work well anyway.